book/forms.rst

```diff
@@ -1021,6 +1021,29 @@ to the ``form()`` or the ``form_start()`` helper:
     a PUT, PATCH or DELETE request. Read the cookbook chapter
     ":doc:`/cookbook/routing/method_parameters`" for more information.
 
+.. note::
+
+  Now that the ``createForm()`` method requires the FQCN, you can not longer pass in ``__construct()`` arguments. To get 
```

Comment:
In my humble opinion yes, but please wait for some reviews from the team.

I guess passing options to the constructor is a bad idea, only helper instances or deep shared config should.
Think of a same form type used many times with different options but same dependencies.
If you take a look at the internal types using dependencies, you'll see that only `ChoiceType` and `EntityType` uses their constructor for the choice factory and loader (btw that's why all the others have been deprecated as services in 3.1).
I'm not sure but I don't think that any internal form types had to be refactored after this BC break.